src/cron-gen.component.js

```diff
@@ -31,15 +31,15 @@ const MONTH_LOOKUPS = {
     '12': 'December'
 };
 const SELECT_OPTIONS = {
-    months: [...new Array(12).keys()].map(x => x + 1),
+    months: [...new Array(12)].map((val, idx) => idx + 1),
```

Comment:
I thought that too, but the const in the component are created outside of the class.  Could move those in to the class...